test/system/ClientAppFixture.cpp

```diff
@@ -15,6 +15,17 @@
 #include <boost/thread/thread.hpp>
 #include <boost/test/unit_test.hpp>
 
+namespace {
+    void AddOptions(OptionsDB& db) {
+#ifdef FREEORION_WIN32
+        db.Add<std::string>("misc.server-local-binary.path", UserStringNop("OPTIONS_DB_FREEORIOND_PATH"),        PathToString(GetBinDir() / "freeoriond.exe"));
```

Comment:
should / could this be in ClientNetworking instead of being duplicated here and chmain ?

Reply:
AI client doesn't need it.

Reply:
having it be available when not needed is probably better than the code duplication, I suppose?

Reply:
I need to call it before `InitDirs` and test fixtures call `InitDirs` after `ClientApp` constructed so I have to properly order calls to move it into ClientNetworking

Reply:
OK